### PR TITLE
PT-3772: Add missing translations for livetable columns

### PR DESCRIPTION
**Pull request by veronikaslc**
Done.

### CODE REVIEW

File: components/navigation/api/src/main/resources/ApplicationResources.properties [hunk 1]
```diff
@@ -37,6 +38,35 @@ patient.livetable.maternal_ethnicity=Maternal ethnicity
...
+patient.livetable.multiple_gestation=Multiple gestation
+patient.livetable.fertility_meds=Conception after fertility meds
+patient.livetable.assisted_reproduction_iui=IUI
+patient.livetable.ivf=IVF
+patient.livetable.icsi=ICSI
+patient.livetable.assistedReproduction_surrogacy=Surrogacy
+patient.livetable.assistedReproduction_donoregg=Donor egg
+patient.livetable.assistedReproduction_donorsperm=Donor sperm
+patient.livetable.apgar1=APGAR 1min
+patient.livetable.apgar2=APGAR 5min
```
Comment (marta-) [inline components/navigation/api/src/main/resources/ApplicationResources.properties]: the key should be `patient.livetable.apgar5`

File: components/navigation/api/src/main/resources/ApplicationResources.properties [hunk 2]
```diff
@@ -37,6 +38,35 @@ patient.livetable.maternal_ethnicity=Maternal ethnicity
 patient.livetable.paternal_ethnicity=Paternal ethnicity
 patient.livetable.family_of=Pedigree number
 patient.livetable.miscarriages=Parents with at least 3 miscarriages
+patient.livetable.life_status=Life status
+patient.livetable.indication_for_referal=Indication for referral
```
Comment (marta-) [inline components/navigation/api/src/main/resources/ApplicationResources.properties]: the key should be `patient.livetable.indication_for_referral`

File: components/navigation/api/src/main/resources/ApplicationResources.properties [hunk 3]
```diff
@@ -37,6 +38,35 @@ patient.livetable.maternal_ethnicity=Maternal ethnicity
...
+patient.livetable.family_history=Family history
+patient.livetable.affected_relatives=Other affected relatives
```
Comment (marta-) [inline components/navigation/api/src/main/resources/ApplicationResources.properties]: the key should be `patient.livetable.affectedRelatives`

File: components/navigation/api/src/main/resources/ApplicationResources.properties [hunk 4]
```diff
@@ -37,6 +38,35 @@ patient.livetable.maternal_ethnicity=Maternal ethnicity
...
+patient.livetable.fertility_meds=Conception after fertility meds
+patient.livetable.assisted_reproduction_iui=IUI
+patient.livetable.ivf=IVF
+patient.livetable.icsi=ICSI
+patient.livetable.assistedReproduction_surrogacy=Surrogacy
+patient.livetable.assistedReproduction_donoregg=Donor egg
+patient.livetable.assistedReproduction_donorsperm=Donor sperm
+patient.livetable.apgar1=APGAR 1min
+patient.livetable.apgar2=APGAR 5min
+patient.livetable.penatal_phenotype=Prenatal and Perinatal History
```
Comment (marta-) [inline components/navigation/api/src/main/resources/ApplicationResources.properties]: the key should be `patient.livetable.prenatal_phenotype`

File: components/navigation/api/src/main/resources/ApplicationResources.properties [hunk 5]
```diff
@@ -37,6 +38,35 @@ patient.livetable.maternal_ethnicity=Maternal ethnicity
...
+patient.livetable.prenatal_development=Prenatal development
+patient.livetable.medical_history=Medical history
+patient.livetable.allergies=Allergies
+patient.livetable.NKDA=NKDA
+patient.livetable.reference_genome=Reference genome
+patient.livetable.genetic_notes=Genetic notes
+patient.livetable.diagnosis_notes=Diagnosis notes
+patient.livetable.solved=Case solved
+patient.livetable.solved_pubmed_id=PUBMED id
```
Comment (marta-) [inline components/navigation/api/src/main/resources/ApplicationResources.properties]: the key should be `patient.livetable.solved__pubmed_id`

File: components/navigation/api/src/main/resources/ApplicationResources.properties [hunk 6]
```diff
@@ -37,6 +38,35 @@ patient.livetable.maternal_ethnicity=Maternal ethnicity
...
+patient.livetable.solved_notes=Resolution notes
```
Comment (marta-) [inline components/navigation/api/src/main/resources/ApplicationResources.properties]: the keu should be `patient.livetable.solved__notes`

**Comment by marta-** [comment]
@sashaandjic I don't think anyone will put filename and reports history in the livetable. the other keys are wrong in the submitted file, I added comments for @veronikaslc to the relevant lines.

**Comment by veronikaslc** [comment]
Thank you, @marta- , @sashaandjic  codereview comments addressed.

**Comment by sashaandjic** [comment]
These are still missing. Sorry I didn't report them yesterday.
- patient.livetable.multipleGestation
- patient.livetable.assistedReproduction_fertilityMeds
- patient.livetable.assistedReproduction_iui
- patient.livetable.reports_history

**Comment by marta-** [comment]
@sashaandjic Fixed.

**Pull request merged**